src/AssemblyLoader.cs

```diff
@@ -30,8 +33,10 @@ public class AssemblyLoader : IAssemblyLoader
         private readonly IAssemblyWrapper _assemblyWrapper;
         private readonly IReflectionWrapper _reflectionWrapper;
         private Assembly _targetLibAssembly;
+        public IStepRegistry StepRegistry = new StepRegistry();
```

Comment:
Why is this an instance variable? Does not seem to be used anywhere. Can we create this in `GetStepRegistry`?